index.html

```diff
@@ -584,7 +604,8 @@ <h1>How can I make a button go full width?</h1>
       </div>
     </footer>
 
-    <div id="modal-card" class="modal modal-fx-3dSlit">
+    <!-- TODO: aria-modal should be set to true when modal is opened -->
```

Comment:
We use open logic from 3rd party script `bulma-modal-fx`, so it's just a potential improvement.